A3A/addons/scrt/Common/fn_common_unconsciousEventHandler.sqf

```diff
@@ -57,6 +57,15 @@ if (reviveKitsEnabled) then {
 				[_nearFriendlyUnits select 0] spawn SCRT_fnc_ai_possessFriendlyUnit;
 			};
 		};
+		case DIK_Z: { // GAMERARMY: One time revive if user presses Z
```

Comment:
This requires revive kits to be enabled, which isn't entirely desirable behaviour.

Reply:
to clarify, this code is present in a 
```c
if (reviveKitsEnabled) {
    ...
}
```

block. I think it'd be best to make it not require revive kits to be enabled, but honestly this doesn't really matter.
It's just good I caught that now before testing it, otherwise I'd probably think the whole thing was bugged since it wouldn't work at all for me.

Just to note though, it being tied to reviveKitsEnabled does seem to mean that all the prompts and stuff around the system like the grace period before being downed will show up even when you can't actually self-revive, since I don't think any other part of this system really checks reviveKitsEnabled.